spec/jobs/after_transition_tasks_for_rejected_return_job_spec.rb

```diff
@@ -62,4 +62,18 @@
       end
     end
   end
+
+  describe '#job_object_id' do
+    let(:submission) { create(:efile_submission, :transmitted) }
+    let(:efile_error) { create(:efile_error, code: "IRS-ERROR", expose: true, auto_wait: false, auto_cancel: false) }
+
+    before do
+      submission.transition_to!(:rejected, error_code: efile_error.code)
+    end
+
+    it 'returns the id of the OutgoingEmail record' do
```

Comment:
Confused -- this talks about an `OutgoingEmail record` but really it's looking for an `EfileSubmission` record, right?

Reply:
that's a good ol' copy paste for ya dawg